.jenkins/pipelines/dependencies.Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+#!groovy
+
+pipeline {
+  agent any
+  tools {
+    maven 'apache-maven-latest'
+    // https://wiki.eclipse.org/Jenkins#JDK
+    jdk 'temurin-jdk11-latest'
+  }
+  options {
+    disableConcurrentBuilds(abortPrevious: true)
+  }
+  stages {
+    stage('Check dependency licenses') {
+      steps {
+        sshagent(['github-bot-ssh']) {
+          sh 'echo "Cloning Project"'
+          sh 'git clone git@github.com:eclipse/jkube.git'
+        }
+        dir('jkube') {
+          sh 'echo "Eclipse Dash Tool"'
+          // Eclipse Dash tool retrieves dependencies from submodules which might need artifacts from previous modules
+          sh './mvnw -V -B -e -DskipTests install'
+          sh 'ECLIPSE_DASH_VERSION=1.0.2 ./scripts/eclipse-dash-tool.sh'
```

Comment:
In CircleCI workflow we're executing both commands with `&&` , is there any specific reason we're executing separately here?

Reply:
The Jenkins pipeline won't continue if the step fails. This makes it easier to check with a glance what caused the issue (problem in the project or actual dependency failure)